client/lang/en.json

```diff
@@ -88,6 +92,23 @@
     "toast_error": "Unable to add measurement",
     "button": "Add new measurement"
   },
+  "group_modal": {
+    "remove_button": "Remove image",
+    "name": "Name",
+    "placeholders": {
+      "name": "Group name",
+      "users": "Enter user's emails"
+    },
+    "labels": {
+      "users": "Users"
```

Comment:
I'd put here something like `Invite members` or `Send invitation emails` so that user knows they will be invited by email message